.github/workflows/presumit.yml

```
@@ -0,0 +1,31 @@
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
+    paths-ignore:
+      - '**.md'
+      - 'docs/**'
+  pull_request_review_comment:
+    types: [created]
+    paths-ignore:
```

actionlint (1.7.4)

"paths-ignore" filter is not available for pull_request_review_comment event. it is only for push, pull_request, pull_request_target events

(events)